src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -296,16 +319,16 @@ export default class PhotoField extends React.Component {
           {!this.state.done && this.state.src && <div className="cropper-container-outer">
             <Cropper
               ref="cropper"
+              ready={this.setCropBox}
```

Comment:
Great improvement, @jbalboni, would it make sense to respond to react-cropper issue 59?

Reply:
Yep, I added a comment.